core/include/kernel/rpc_io_i2c.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2020 Foundries Ltd <jorge@foundries.io>
+ */
+
+#ifndef __RPC_IO_I2C_H
+#define __RPC_IO_I2C_H
+
+#include <optee_rpc_cmd.h>
+#include <tee_api_types.h>
+
+/* I2C master transfer mode */
+enum rpc_i2c_mode {
+	RPC_I2C_MODE_WRITE = OPTEE_MSG_RPC_CMD_I2C_TRANSFER_WR,
+	RPC_I2C_MODE_READ = OPTEE_MSG_RPC_CMD_I2C_TRANSFER_RD,
+};
+
+/* I2C master transfer control flags */
+#define RPC_I2C_FLAGS_TEN_BIT	OPTEE_MSG_RPC_CMD_I2C_FLAGS_TEN_BIT
+
+/*
+ * The bus identifier defines an implicit ABI with the REE.
+ * Using this service to access I2C slaves on REE dynamically assigned buses is
```

Comment:
It looks to me that this describes the way Linux kernel currently assigns bus indices, no?

Reply:
I dont see it that way -  but I also think that this comment (and many others) are ultimately unnecessary. In any case let me know if I can remove these in particular or just tell me what you want written and I'll just paste.

Reply:
I reworded it a little. let me know if I should remove it.